plugin-cgroupv2/src/k8s/utils.rs

```diff
@@ -34,9 +37,29 @@ pub struct CgroupV2MetricFile {
     pub node: String,
 }
 
-/// Check if a specific file is a dir. Used to know if cgroup v2 are used
-pub fn is_accessible_dir(path: &Path) -> bool {
-    path.is_dir()
+/// Check if a specific file is a dir. Used to know if cgroup v2 are used.
+///
+/// # Return
+///
+/// - Boolean error ocurres when path of a file or directory is not available
+/// - Error ocurres when an other kind of issue appears on the path
```

Comment:
```suggestion
/// # Return value
///
/// Returns `Ok(true)` if it can be verified that `path` is a directory, and `Ok(false)` if it can be verified that it is not a directory.
/// Returns an error if the path metadata cannot be obtained.
```